src/statsig/constants.ts

```diff
@@ -102,4 +102,15 @@ export const DynamicConfigs = {
       showTransfers: [networkConfig.defaultNetworkId],
     },
   },
+  [StatsigDynamicConfigs.DAPP_WEBVIEW_CONFIG]: {
+    configName: StatsigDynamicConfigs.DAPP_WEBVIEW_CONFIG,
+    defaultValues: {
+      ios: {
+        disableMediaPlaybackRequiresUserAction: [] as string[],
+      },
+      android: {
+        disableMediaPlaybackRequiresUserAction: [] as string[],
```

Comment:
Could we control ios vs android directly in statsig instead?
So we have a simple config to read.

Reply:
ah, TY for suggesting this. i forgot we could do this 😅 updated

Reply:
the config is here https://console.statsig.com/4plizaPmWwPL21ASV4QAO0/dynamic_configs/dapp_webview_config